contributors/devel/sig-scalability/kubemark-setup-guide.md

```diff
@@ -0,0 +1,60 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster, both inside and outside gce.
+
+## Precondition
+You need two k8s clusters(cluster one and cluster two) to set up one kubemark cluster. 
```

Comment:
Here and below, instead of just saying "cluster one" and "cluster two", can you use the terms "kubemark cluster" and "external cluster" respectively to resonate with this guide - https://github.com/kubernetes/community/blob/master/contributors/devel/sig-scalability/kubemark-guide.md

Reply:
Done